src/connectors/snowflake/trulens/connectors/snowflake/utils/server_side_evaluation_artifacts.py

```diff
@@ -79,6 +89,7 @@ def _set_up_stored_procedure(self) -> None:
                 LANGUAGE PYTHON
                 RUNTIME_VERSION = '3.11'
                 PACKAGES = (
+                    {trulens_packages}
                     -- TODO(dkurokawa): get these package versions automatically.
```

Comment:
is this TODO still relevant?

Reply:
Yeah I still have to do that at some point (though it's not entirely clear how that could be done yet).